#	--- libdv-2006-01-15/ChangeLog	2006-01-15 22:23:15.000000000 +0100
#	+++ libdv-2006-01-15-bk/ChangeLog	2006-02-13 20:36:58.000000000 +0100
#	@@ -1,3 +1,7 @@
#	+2006-02-13  Bernhard Kaindl <bk@suse.de>
#	+        * audio.c: reduced excessive warning messages which caused
#	+          interrupted play on many streams due to excessive output
#	+
#	 2006-01-15  Dan Dennedy <dan@dennedy.org>
#		* encode.c: apply patch from Burkhard Plaum to use gcc
#		  destructor attribute to invoke dv_cleanup() when unloading
--- libdv-2006-01-15/libdv/audio.c	2006-02-13 18:45:35.000000000 +0100
+++ libdv-2006-01-15-bk/libdv/audio.c	2006-02-13 20:35:21.000000000 +0100
@@ -613,7 +613,7 @@ dv_decode_audio_block(dv_audio_t *dv_aud
      * check if some or all samples in block failed
      */
     if (full_failure) {
-      if (dv_audio -> error_log) {
+      if (dv_audio -> error_log && full_failure != 1) {
         if (dv_get_timestamp (dv_audio -> dv_decoder, err_msg1) &&
             dv_get_recording_datetime (dv_audio -> dv_decoder, err_msg2)) {
         fprintf (dv_audio -> error_log,
--- libdv-2006-01-15/libdv/dv.c	2004-10-20 05:49:24.000000000 +0200
+++ libdv-2006-01-15-bk/libdv/dv.c	2006-02-13 20:35:45.000000000 +0100
@@ -569,7 +569,7 @@ dv_decode_full_audio(dv_decoder_t *dv, c
   } /* for */
 
   if (dv -> audio -> sample_failure) {
-    if (dv -> audio -> error_log) {
+    if (dv -> audio -> error_log && dv -> audio -> block_failure) {
       fprintf (dv -> audio -> error_log,
                "# audio block/sample failure for %d blocks, %d samples of %d\n",
                dv -> audio -> block_failure,
